yt/frontends/adaptahop/data_structures.py

```diff
@@ -35,9 +37,24 @@
 from .fields import \
     AdaptaHOPFieldInfo
 
+class AdaptaHOPParticleIndex(ParticleIndex):
```

Comment:
@cphyc, this was `HaloCatalogParticleIndex`, but I realized that it wasn't being used anywhere else so I moved it here. Let me know if you'd like me to move it back. It's also possible that this can work simply as `ParticleIndex`, but I didn't test that.

Reply:
I'm fine with this!